contents/docs/error-tracking/releases.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: Releases
+showTitle: true
+sidebar: Docs
+---
+
+Releases describe a single deployment: the artefacts you uploaded, the version that produced them, and any build metadata PostHog can infer.
+
+## Creating releases
```

Comment:
I'd add a link to the upload source map docs, since we'll be adding this info there.